webui/react/src/components/UserSettings.tsx

```diff
@@ -1,4 +1,5 @@
 import { Space } from 'antd';
+import { partial, type } from 'io-ts';
```

Comment:
Idiomatic io-ts import is `import * as t from 'io-ts'`.

Reply:
✅